RELEASE/scripts/autoscend/paths/wildfire.ash

```diff
@@ -2,3 +2,462 @@ boolean in_wildfire()
 {
 	return auto_my_path() == "Wildfire";
 }
+
+void wildfire_initializeSettings()
+{
+	if(!in_wildfire())
+	{
+		return;
+	}
+	set_property("auto_wandOfNagamar", false);		//wand not used in this path
+	set_property("auto_getBeehive", true);			//fire cannot be reduced from 5 in tower making the fight too difficult without beehive
+}
+
+boolean wildfire_groar_check()
+{
+	//Prepare to fight [Groar, Except Hot] in wildfire path. Also check if we are ready for the fight.
+	//Return true if we are not ready and should delay this fight
+	//[Mist-Shrouded Peak] cannot reduced fire level from 5. Take 20-25% maxHP hot passive dmg per round. always lose initiative
+	if(!in_wildfire())
+	{
+		return false;	//since we are not in wildfire, we are considered "ready" so we do not block the quest in other paths
+	}
+	if(my_level() < 13)
+	{
+		return true;	//we want to delay this until we are level 13. The more stats we have the easier the fight is.
+	}
+	if(!acquireHP() || !acquireMP(150))
+	{
+		return true;	//killing groar requires lots of MP and full HP.
+	}
+	setFlavour($element[sleaze]);		//deal extra damage against hot opponent
+	
+	return false;
+}
+
+boolean wildfire_warboss_check()
+{
+	//Prepare to fight [The Big Ignatowicz] or [The Man on Fire] in wildfire path. Also check if we are ready for the fight.
+	//Return true if we are not ready and should delay this fight
+	//[Hippy Camp] and [Frat House] cannot reduced fire level from 5. Take 20-25% maxHP hot passive dmg per round. always lose initiative
+	if(!in_wildfire())
+	{
+		return false;	//since we are not in wildfire, we are considered "ready" so we do not block the quest in other paths
+	}
+	if(!acquireHP() || !acquireMP(150))
+	{
+		return true;	//killing warboss requires lots of MP and full HP.
+	}
+	setFlavour($element[sleaze]);		//deal extra damage against hot opponent
+	
+	return false;
+}
+
+boolean LX_wildfire_calculateTheUniverse()
+{
+	//in wildfire calculate the universe always summons in a fire 5 zone which 100% burns all dropped items. unless conditional drops
+	//yellow ray items still burn up. the only exception is [use the force] because it brings you to a noncombat to give you the items
+	if(!in_wildfire())
+	{
+		return false;
+	}
+	if(my_mp() < mp_cost($skill[Calculate the Universe]))
+	{
+		return false;
+	}
+	
+	if(!possessOutfit("Frat Warrior Fatigues") && auto_warSide() == "fratboy")
+	{
+		if(doNumberology("battlefield", false) != -1 && auto_saberChargesAvailable() > 0)
+		{
+			autoEquip($slot[weapon], $item[Fourth of May cosplay saber]);
+			return (doNumberology("battlefield") != -1);
+		}
+		return false;	//we want 151 and can get it in general. but not right now. so save it for later
+	}
+	
+	doNumberology("adventures3");
```

Comment:
this will never get hit until you have the fatigues if you don't have a saber. This should be fixable by moving the saber check to the possessOutfit if check.

Reply:
@Rinn fixed